frontends/election-manager/src/contexts/app_context.ts

```diff
@@ -48,6 +49,8 @@ export interface AppContextInterface {
   updateExternalTally: (
     newExternalTally: FullElectionExternalTally
   ) => Promise<void>;
+  manualTallyVotingMethod: VotingMethod;
+  setManualTallyVotingMethod: (votingMethod: VotingMethod) => void;
```

Comment:
I moved the default voting method for manual data up into app state. This means that, unfortunately, that data can be in two places - in the app state and as the voting method for the external tally. 

The original set up of having multiple entry points to create the manual tally, and also creating an empty tally when the voting method was toggled, was not ideal.